packages/flutter_driver/test/src/real_tests/extension_test.dart

```diff
@@ -473,7 +473,7 @@ void main() {
     });
 
     testWidgets('works when semantics are enabled', (WidgetTester tester) async {
-      final SemanticsHandle semantics = RendererBinding.instance.pipelineOwner.ensureSemantics();
+      final SemanticsHandle semantics = tester.ensureSemantics();
```

Comment:
I assume this change is to verify the new counting logic work correctly, should we create a new test instead of modifying the existing one?

Reply:
Added a test. Turns out, the logic was previously untested and contained a bug. Now some more tests that fail to clean up their semantics handle are failing. I'll have to look into that tomorrow.

Reply:
Done. Could you take another look at the new tests and the fixes?